### PR TITLE
Prepare release r1.1

### PR DESCRIPTION
**Pull request by jgarciahospital**
#### What type of PR is this?

Add one of the following kinds:

* documentation
* subproject management

#### What this PR does / why we need it:

Prepare the release 1.1 with the required changes to support meta guidelines

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,10 +2,42 @@
...
+
+The API definition(s) are based on
+* Commonalities v0.4.0-rc.1
+* Identity and Consent Management v0.2.0-rc.1
+
+
+## population-density-data v0.1.1-rc.1
+
+**population-density-data v0.1.1-rc.1 is the 1st release candidate of the version 0.1.1**
+
```
Comment (hdamker) [inline CHANGELOG.md:24]:
```suggestion

v0.1.1 will be a patch release of v0.1.0 which only changes the info object in alignment with Commonalities 0.4.0.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,10 +2,42 @@
...
+- API definition **with inline documentation**:
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/PopulationDensityData/blob/release-v0.1.1-rc1.1/code/API_definitions/population-density-data.yaml)
+  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/PopulationDensityData/release-v0.1.1-rc1.1/code/API_definitions/population-density-data.yaml&nocors)
+  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/PopulationDensityData/release-v0.1.1-rc1.1/code/API_definitions/population-density-data.yaml)
+
+### Added
+* Include  `x-camara-commonalities` by @jgarciahospital in https://github.com/camaraproject/PopulationDensityData/pull/
```
Comment (hdamker) [inline CHANGELOG.md:31]:
```suggestion
* Include  `x-camara-commonalities` by @jgarciahospital in https://github.com/camaraproject/PopulationDensityData/pull/41
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,10 +2,42 @@
...
+
+### Changed
+
+### Fixed
+
+### Removed
+* Remove `terms of service` and `contact` by @jgarciahospital in https://github.com/camaraproject/PopulationDensityData/pull/
```
Comment (hdamker) [inline CHANGELOG.md:38]:
```suggestion
* Remove `terms of service` and `contact` by @jgarciahospital in https://github.com/camaraproject/PopulationDensityData/pull/41
```

File: README.md [hunk 4]
```diff
@@ -30,11 +30,11 @@ Repository to describe, develop, document and test the Population Density Data A
...
-* Release 0.1.0 of the API family is available within the [release-v0.1.0](https://github.com/camaraproject/PopulationDensityData/tree/release-v0.1.0) branch.
-- 0.1.0 Population Density Data API definition **with inline documentation**:
-  - OpenAPI [YAML spec file](https://github.com/camaraproject/PopulationDensityData/blob/release-v0.1.0/code/API_definitions/population-density-data.yaml)
-  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/PopulationDensityData/release-v0.1.0/code/API_definitions/population-density-data.yaml&nocors)
-  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/PopulationDensityData/release-v0.1.0/code/API_definitions/population-density-data.yaml)
+* Release 0.1.1-rc1.1 of the API family is available within the [release-v0.1.1-rc1.1](https://github.com/camaraproject/PopulationDensityData/tree/release-v0.1.1-rc1.1) branch.
```
Comment (hdamker) [inline README.md:33]:
```suggestion
* The pre-release r1.1 contains the version 0.1.1-rc.1 of the Population Density Data API. It is the first release candidate of the patch release v0.1.1. The release tag is [r1.1](https://github.com/camaraproject/PopulationDensityData/tree/r1.1).
```

There is no mandatory text therefore this is just a suggestion. 
Important is just:
* we don't use/create release branches any more
* The release tag is r1.1
* The API Version is v0.1.1-rc.1, not v0.1.1-rc1.1

File: README.md [hunk 5]
```diff
@@ -30,11 +30,11 @@ Repository to describe, develop, document and test the Population Density Data A
...
+- 0.1.1-rc1.1 Population Density Data API definition **with inline documentation**:
```
Comment (hdamker) [inline README.md:34]:
```suggestion
- 0.1.1-rc.1 Population Density Data API definition **with inline documentation**:
```

File: README.md [hunk 6]
```diff
@@ -30,11 +30,11 @@ Repository to describe, develop, document and test the Population Density Data A
...
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/PopulationDensityData/blob/release-v0.1.1-rc1.1/code/API_definitions/population-density-data.yaml)
```
Comment (hdamker) [inline README.md:35]:
```suggestion
  - OpenAPI [YAML spec file](https://github.com/camaraproject/PopulationDensityData/blob/r1.1/code/API_definitions/population-density-data.yaml)
```

File: README.md [hunk 7]
```diff
@@ -30,11 +30,11 @@ Repository to describe, develop, document and test the Population Density Data A
...
+  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/PopulationDensityData/release-v0.1.1-rc1.1/code/API_definitions/population-density-data.yaml&nocors)
```
Comment (hdamker) [inline README.md:36]:
```suggestion
  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/PopulationDensityData/r1.1/code/API_definitions/population-density-data.yaml&nocors)
```

File: README.md [hunk 8]
```diff
@@ -30,11 +30,11 @@ Repository to describe, develop, document and test the Population Density Data A
...
+  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/PopulationDensityData/release-v0.1.1-rc1.1/code/API_definitions/population-density-data.yaml)
```
Comment (hdamker) [inline README.md:37]:
```suggestion
  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/PopulationDensityData/r1.1/code/API_definitions/population-density-data.yaml)
```

File: code/API_definitions/population-density-data.yaml [hunk 9]
```diff
@@ -110,18 +110,17 @@ info:
...
     (FAQs will be added in a later version of the documentation)
-  termsOfService: http://swagger.io/terms/
-  contact:
-    email: project-email@sample.com
+    
   license:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0.html
-  version: 0.1.0
+  version: 0.1.1-rc1.1
```
Comment (hdamker) [inline code/API_definitions/population-density-data.yaml:117]: Two options here from my point of view:

v0.1.1-rc.1 (first release candidate of v0.1.1)
or
v0.1.1-rc.3 (as there were theoretically already two release candidates of v0.1.0 and only with continuing the number we would get a unique url for this release candidate)

Or directly v0.1.1 as this is only a patch release of v0.1.0 with minor changes and no release candidate is needed ... but then also several other place need to be changed accordingly. 


```suggestion
  version: 0.1.1-rc.1

```

File: code/API_definitions/population-density-data.yaml [hunk 10]
```diff
@@ -110,18 +110,17 @@ info:
...
+  x-camara-commonalities: 0.4.0
 externalDocs:
   description: Product documentation at Camara.
   url: https://github.com/camaraproject/
 servers:
-  - url: '{apiRoot}/population-density-data/v0.1'
+  - url: '{apiRoot}/population-density-data/v0.1rc1'
```
Comment (hdamker) [inline code/API_definitions/population-density-data.yaml:123]: Correct if the version will be v0.1.1-rc.1

File: code/Population-Density-Data-API-Readiness-Checklist.md
Comment (hdamker) [inline code/Population-Density-Data-API-Readiness-Checklist.md:1]: The [release management documentation](https://wiki.camaraproject.org/display/CAM/API+Release+Process#APIReleaseProcess-APIreadinesschecklist) said initially that these file should go into /code, but it was changed to /documentation/API_documentation (the location were also the previous checklists were located).

File: code/Population-Density-Data-API-Readiness-Checklist.md [hunk 11]
```diff
@@ -0,0 +1,18 @@
...
+|  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |  Y   |      |
+|  4 | API versioning convention applied            |   M   |         M         |    M    |    M   |  Y   |      |
+|  5 | API documentation                            |   M   |         M         |    M    |    M   |  Y   | Embed documentation into API spec - [link](/code/API_definitions/number_verification.yaml) |
+|  6 | User stories                                 |   O   |         O         |    O    |    M   |  N   |      |
+|  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |  Pending   |  |
+|  8 | Enhanced API test cases & documentation      |   O   |         O         |    O    |    M   |   N  |  |
+|  9 | Test result statement                        |   O   |         O         |    O    |    M   |   N  |  |
+| 10 | API release numbering convention applied     |   M   |         M         |    M    |    M   |   Y  |      |
+| 11 | Change log updated                           |   M   |         M         |    M    |    M   |   Y  | [link](https://github.com/camaraproject/NumberVerification/pull/121) |
```
Comment (hdamker) [inline code/Population-Density-Data-API-Readiness-Checklist.md:17]:
```suggestion
| 11 | Change log updated                           |   M   |         M         |    M    |    M   |   Y  | [link](/CHANGELOG.md) |
```

**Comment by hdamker** [comment]
@jgarciahospital See my comments above. I assumed that this will be indeed a release candidate of v0.1.1. If the changes are done, please add @camaraproject/release-management_maintainers as reviewer.

File: code/Population-Density-Data-API-Readiness-Checklist.md
Comment (jgarciahospital) [reply code/Population-Density-Data-API-Readiness-Checklist.md:1]: copy/past issues. Fixed.

**Comment by hdamker** [comment]
API definition:
* update the Authorization and Authentication text with the latest updates from ICM:
  * it was in general changed in 0.2.0
  * there was a last minute update to avoid the link into ICM main branch, see https://github.com/camaraproject/IdentityAndConsentManagement/pull/186, which will be part of the second release candidate of ICM
* change the link within line 108 accordingly, as links into 'main' branch will break at some point of time.

**Comment by jgarciahospital** [comment]
@hdamker ready for new revision of this fields

> API definition:
> 
> update the Authorization and Authentication text with the latest updates from ICM:
> it was in general changed in 0.2.0
> there was a last minute update to avoid the link into ICM main branch, see https://github.com/camaraproject/IdentityAndConsentManagement/pull/186, which will be part of the second release candidate of ICM
> change the link within line 108 accordingly, as links into 'main' branch will break at some point of time.

Also added #42 for documentation and included in changelog, as agreed in followup meeting#12

File: code/API_definitions/population-density-data.yaml [hunk 12]
```diff
@@ -94,37 +94,30 @@ info:
...
+
+    Which specific authorization flows are to be used will be determined during onboarding process, happening between the API Client and the Telco Operator exposing the API, taking into account the declared purpose for accessing the API, while also being subject to the prevailing legal framework dictated by local legislation.
+
+    It is important to remark that in cases where personal user data is processed by the API, and users can exercise their rights through mechanisms such as opt-in and/or opt-out, the use of 3-legged access tokens becomes mandatory. This measure ensures that the API remains in strict compliance with user privacy preferences and regulatory obligations, upholding the principles of transparency and user-centric data control.
+
+    Population Density Data API ensures the usage of anonymized information and do not treat personal data neither as input nor output. 
+    Therefore, the access to Population Density Data API is defined as Client Credentials - 2-legged. Please refer to Identify and Consent Management (https://github.com/camaraproject/IdentityAndConsentManagement/) for the last detailed specification of this authentication/authorization flow.
```
Comment (hdamker) [inline code/API_definitions/population-density-data.yaml:104]: I suppose latest, not last is meant here:

```suggestion
    Therefore, the access to Population Density Data API is defined as Client Credentials - 2-legged. Please refer to Identify and Consent Management (https://github.com/camaraproject/IdentityAndConsentManagement/) for the latest detailed specification of this authentication/authorization flow.

```

**Review by hdamker — APPROVED** [review]
Looks good from Release Management. Please proceed with the release.

**Comment by hdamker** [comment]
> Thanks @hdamker. As no other codeOwner active (holidays), may you validate the PR?

I can bypass the rule protection and merge, but would be good if at least one further Sub Project participant would approve the PR, @gregory1g maybe?

File: code/API_definitions/population-density-data.yaml [hunk 13]
```diff
@@ -94,37 +94,31 @@ info:
...
+    Population Density Data API ensures the usage of anonymized information and do not treat personal data neither as input nor output. 
+    Therefore, the access to Population Density Data API is defined as Client Credentials - 2-legged. Please refer to Identify and Consent Management (https://github.com/camaraproject/IdentityAndConsentManagement/) for the latest detailed specification of this authentication/authorization flow.
+
 
     # Further info and support
 
-  termsOfService: http://swagger.io/terms/
-  contact:
-    email: project-email@sample.com
+    (FAQs will be added in a later version of the documentation)
```
Comment (gregory1g) [inline code/API_definitions/population-density-data.yaml:109]: why add "TBD" entry to the released version of the spec?

**Review by gregory1g — APPROVED** [review]

File: code/API_definitions/population-density-data.yaml (hunk 13 above)
Comment (hdamker) [reply code/API_definitions/population-density-data.yaml:109]: This line was copied over from the very first API documentations. It was never mandatory, even not in the outdated documentation template (which will now get [deprecated](https://github.com/camaraproject/Commonalities/pull/261)). So you can delete this line.

Comment (gregory1g) [reply code/API_definitions/population-density-data.yaml:109]: I have removed it PR #42 . And now it is added back.

**Comment by jgarciahospital** [comment]
@hdamker ready now?

**Pull request merged**